pfcpiface/config.go

```diff
@@ -1,5 +1,7 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright 2022-present Open Networking Foundation
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2020-2022 Intel Corporation
```

Comment:
Why this additional copyright and license if the only change in this file is adding the following text?
`"cndp":      {},`

Reply:
I can remove the copyright.